neovim/ui/gtk_ui.py

```diff
@@ -124,6 +124,11 @@ def wrapper():
             self._screen_invalid()
         GObject.idle_add(wrapper)
 
+    def _nvim_update_menu( self, *args ):
```

Comment:
this should be placed among the other `_nvim_` event handlers. Also just `pass` is fine here, compare `_nvim_visual_bell` (we won't expect all guis to respond to all ui updates)